src/app/mypage/team/page.tsx

```
@@ -0,0 +1,222 @@
+        <Text variant={'heading3'} color={'primary4'}>
+          💜 우리 팀을 소개할게요! 💜
+        </Text>
+      </div>
+
+      {/* 프로젝트 소개 섹션 */}
+      <section className="w-full px-6 py-16 bg-gradient-to-b from-[#f3f0ff] to-white flex flex-col items-center gap-5 text-center">
+        <Text variant={'heading3'} color={'primary4'}>
+          Don’t Worry – 걱정이 많아도 괜찮아!
+        </Text>
+        <Text variant={'body1'} color={'label-neutral'}>
+          <span>"밤마다 걱정이 머리를 떠나지 않나요?"</span>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)